File: app/src/main/java/com/sorgs/app/MainActivity.java
package com.sorgs.app;

import android.view.View;

import com.sorgs.baseproject.base.JavaBaseActivity;
import com.sorgs.baseproject.utils.ToastUtils;

/**
 * @author sorgs.li
 */
public class MainActivity extends JavaBaseActivity {


    @Override
    protected int initLayoutId() {
        return R.layout.activity_main;
    }

    @Override
    protected void initData() {
        super.initData();

        findViewById(R.id.tv_hello).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ToastUtils.INSTANCE.showShort("hello");
            }
        });
    }
}

File: app/src/main/java/com/sorgs/app/MainActivity.kt
package com.sorgs.app

import android.os.Bundle
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentTransaction
import com.sorgs.baseproject.base.JavaBaseActivity


/**
 * @author sorgs
 */
class MainActivity : JavaBaseActivity() {
    override fun initLayoutId(): Int {
        return R.layout.activity_main
    }

    //override fun staticTopViewId() = R.id.ll_test

    //override fun isNeedShowStatusBar(): Boolean = true


    override fun initView(savedInstanceState: Bundle?) {
        super.initView(savedInstanceState)
        val fragment1 = TestFragment()
        val manager: FragmentManager = supportFragmentManager
        val transaction: FragmentTransaction = manager.beginTransaction()
        transaction.add(R.id.ll_test, fragment1).commit()
    }
}